app/javascript/legacy_react/src/lib/utils.spec.ts

```diff
@@ -0,0 +1,29 @@
+import * as Utils from './utils';
```

Comment:
Please add the license header on this file. You can find it at the top of any of the other TS files.

Reply:
No problem!